staker/staker_test.gno

```diff
@@ -157,38 +158,21 @@ func TestWithdrawAmount(t *testing.T) {
 	}
 }
 
-func TestWithdrawAmount_CallerNotDelegate(t *testing.T) {
-	token := newMockGRC20("Test Token", "TST", 42)
-	Init(token)
-
-	delegate := std.Address("delegate")
-	recipient := std.Address("recipient")
-	stakeAmount := u256.NewUint(100)
-	withdrawAmount := u256.NewUint(50)
-
-	caller := std.GetOrigCaller()
-	token.balances[caller] = 1000
-	err := StakeAmount(delegate, stakeAmount) // must std.GetOrigCaller() != delegate
-	if err != nil {
-		t.Fatalf("Failed to stake initial amount: %v", err)
-	}
-}
-
 func TestGetAverageDelegated(t *testing.T) {
 	token := newMockGRC20("Test Token", "TST", 42)
 	Init(token)
 
-	delegate := std.Address("delegate")
-	stakeAmount := u256.NewUint(0)
+	delegate := "delegate"
+	delegateAddr := std.Address(delegate)
```

Comment:
testutils to generate valid address

Reply:
fixed https://github.com/gnoswap-labs/governance/pull/14/commits/99ef445de0ad28686998bd36ff249a3e9dc686c1